Math/PermutationSequence.swift

```diff
@@ -0,0 +1,39 @@
+/**
+ * Question Link: https://leetcode.com/problems/permutation-sequence/
+ * Primary idea: Iterate and change the array from last to the first
+ *
+ * Time Complexity: O(n^2), Space Complexity: O(1)
+ */
+
+class PermutationSequence {
+	func getPermutation(_ n: Int, _ k: Int) -> String {
```

Comment:
Your indention looks a little weird. Seems the indent width is 7 spaces? Could you change it to 2 or 4 and reformat your code? Thanks!

Reply:
Wired problem. It looks good to me in VSCode. I fixed this and will check my IDE setting later.